packages/mwp-tracking-plugin/src/util/avro.js

```diff
@@ -81,8 +81,12 @@ const getLogAWSKinesis = (): (string => Promise<void>) => {
 		});
 
 		kinesis.putRecord(options, function(err, data) {
-			if (err) console.log(err, err.stack);
-			else console.log(data);
+			if (err) {
+				console.log("Error sending message to Kinesis: "+ serializedRecord);
+				console.log(err, err.stack);
+			} else {
+				console.log("Kinesis recieved message: " + serializedRecord);
```

Comment:
I put this line in so we can test that the serialized record written out to cw is in the format that we expect/we have all the data to reprocess. I'll do a follow-up pr to reduce the volume of logs in the case when we successfully write to flume